### ISSUE TITLE
에브리타임 어플 광고차단

### ISSUE THREAD

**Issue by rewrite15** [yous/YousList#67]
여기다 올리는게 맞는지 모르겠지만... 아니면 지우겠습니다.

에브리타임 어플 내의 광고가 차단이 안됩니다. 게시판 목록 들어가면 어플 하단에 광고창이 생깁니다.

**Comment by yous** [comment]
네, AdGuard 사용하시는 거죠? 안드로이드 버전도 알 수 있을까요? 안드로이드 누가부터 HTTPS 차단 방식이 달라져서 광고가 보일 수도 있습니다.

**Comment by rewrite15** [comment]
아 지금 8.0.0 사용중입니다
버전에따라 막힌게 있나보네요 답변감사드립니다

**Comment by yous** [comment]
네, 확인 후에 알려드리도록 하겠습니다.
